Tests/StyleguideTests/__Snapshots__/StyleguideTests/testDesignSystem.1.pretty.css

```diff
@@ -231,13 +231,13 @@
 }
 
 .col-m-4 {
-  flex-basis : 33.3333333333333%;
-  max-width  : 33.3333333333333%;
+  flex-basis : 33.333333333333336%;
+  max-width  : 33.333333333333336%;
 }
 
 .col-m-5 {
-  flex-basis : 41.6666666666667%;
-  max-width  : 41.6666666666667%;
+  flex-basis : 41.666666666666664%;
+  max-width  : 41.666666666666664%;
```

Comment:
ughghgh looking at the linux failures:

```diff
− flex-basis : 41.666666666666664%;
− max-width  : 41.666666666666664%;
+ flex-basis : 41.6666666666667%;
+ max-width  : 41.6666666666667%;
```

Reply:
Yeah we got a buncha work before we can merge this. Floating point precision we need to manually truncate at rendering. And we gotta figure out key-value ordering :(